### DISCUSSION TITLE
callback clicked-setting() & clicked-account() in SMenu doesnt exist

### DISCUSSION THREAD

**Discussion by manish-ach** [syf20020816/SurrealismUI#46] [Q&A]
The callbacks for SMenu are listed in the wiki, though when looking over around menu.slint and global callbacks in the files i cant seem to find them. I am also a newbie going through the wiki and documentations. Though i cant see any customization property for the SMenu. I did find menu-data but i dont know how to use it and cant find any documentation on it too. Is the SMenu still a work in progress or is the documentation for it is not updates.
I do really need help using the SMenu widget and how to edit it.

**Comment by syf20020816** [comment]
See here: https://surrealism-all.github.io/SurrealismUI.github.io/components/nav/menu.html
the wiki is old, you should read this book

**Comment by syf20020816** [comment]
Seems I forget update the callbacks
```slint
callback clicked-account() : run if you click account icon
callback clicked-setting() : run if you click setting icon
```
these callbacks are merge into change (callback)

**Comment by syf20020816** [comment]
What's more you can see this video to learn how to use menu:
https://www.bilibili.com/video/BV14U411f7LX?spm_id_from=333.788.videopod.episodes&vd_source=9c2ae08297163e79152407c969f04717&p=34
